Tests/Qt/QmlApiTest.cpp

```diff
@@ -85,6 +86,66 @@ class QmlApiTest : public QObject
       }
     }
 
+
+    void test_logFilter()
+    {
+      int argc{}; char** argv{};
+      QCoreApplication app(argc, argv);
+      ossia::context context;
+      QQmlEngine engine;
+
+      engine.addImportPath(QDir().absolutePath() + "/testdata/qml");
+      engine.addPluginPath(QDir().absolutePath() + "/testdata/qml");
+
+      {
+        QQmlComponent component(&engine);
+        component.setData(R"_(
+                          import Ossia 1.0 as Ossia
+                          import QtQuick 2.5
+                          Item{
+                          Connections{
+                          target: Ossia.Logger
+                          onFilteredLog :
+                          {
+                               console.log('filtered: ' + msg + ' '+ type+ ' '+fileName + ' '+line );
+                          }
+
+                          }
+                          Component.onCompleted:{
+                          console.log("test")
+
+                          Ossia.Logger.logFilter= ["cameraBin","touchpoint"];
+                          Ossia.Logger.loggerHost = 'ws://127.0.0.1:1337';
+                          Ossia.Logger.logQtMessages = true;
+
+                          Ossia.Logger.startHeartbeat({});
+
+
+                          console.log("avant")
+                          console.log('touchpoint')
+
+                          console.log("touchy")
+                          console.log("cameraBin");
+                          console.log("autre")
+                          }
+
+
+
+                          }
+                          )_", QUrl{});
+
+        qDebug() << component.errorString();
+        QVERIFY(component.errors().empty());
```

Comment:
could it be possible to add a QVERIFY to check that the message was correctly logged or not ?